### ISSUE TITLE
ld: unknown option: -soname clang: error: linker command failed with exit code 1 (use -v to see invocation)

### ISSUE THREAD

**Issue by saurabhaec** [CopernicaMarketingSoftware/PHP-CPP#368]
get an error on mac pro during installation of PHP-CPP

sudo make  

ld: unknown option: -soname
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make: *** [libphpcpp.so.2.0.0] Error 1

**Comment by sjinks** [comment]
Just replace

```
-Wl,-soname,libphpcpp.so.$(SONAME)
```

with

```
-Wl,-install_name,libphpcpp.so.$(SONAME)
```

in Makefile.

**Comment by mstaack** [comment]
@sjinks  thanks this solved my error also!!

**Comment by mvdwerve** [comment]
I think this is solved with PR #391
